test-case/check-pause-release-suspend-resume.sh

```diff
@@ -145,8 +150,15 @@ expect {
             exit 1
         }
 
+        #get the type of platform
+        set retval [catch { exec uname -a | grep -o glk } platform]
+
         #enter suspend-resume cycle once per pause instance
-        set retval [catch { exec bash check-suspend-resume.sh -l 1 } msg]
+        if {[string equal \$platform "glk"] != 0} {
+            set retval [catch { exec bash $SUSPEND_RESUME -T s2idle} msg]
+        } else {
+            set retval [catch { exec bash $SUSPEND_RESUME} msg]
+        }
```

Comment:
Finally, forward it to check-suspend-resume:
```suggestion
        set retval [catch { exec bash $CASEDIR/check-suspend-resume.sh -l 1 -T $type } msg]
```